Resources/Prototypes/_Scp/Recipes/Crafting/Graphs/roomba.yml

```
@@ -0,0 +1,32 @@
+- type: constructionGraph
+  id: RoombaConstruct
+  start: start
+  graph:
+  - node: start
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)